axon-server/administration/backup-and-messaging-only-nodes.md

```diff
@@ -4,131 +4,172 @@
+They do not participate in transactions and will clearly never become the leader for a replication-group.
 
-There are three possible ways to assign the MESSAGING\_ONLY role to a node within a context:
+There are three possible ways to assign the MESSAGING\_ONLY role to a node within a replication group:
 
-A\) The Axon Server EE UI Console. Navigate to the Contexts icon on the navigation menu of the console which will open up the context maintenance screen. The nodes can be added as a MESSAGING\_ONLY role within a context.
+A\) The Axon Server EE UI Console. Navigate to the Replication Groups icon on the navigation menu of the console which will open up the replication group maintenance screen. The nodes can be added as a MESSAGING\_ONLY role within a replication group.
 
-B\) The _add-node-to-context_ command with the role option as MESSAGING\_ONLY
+B\) The _add-node-to-replication-group_ command with the role option as MESSAGING\_ONLY
 
 ```text
-$ java -jar axonserver-cli.jar add-node-to-context  -S http://[node]:[port] -n [node name]‌ -c [context-name] -r MESSAGING_ONLY
+$ java -jar axonserver-cli.jar add-node-to-replication-group  -S http://[node]:[port] -n [node name] -g [replication-group-name] -r MESSAGING_ONLY
 ```
 
 _Mandatory parameters_
 
-* _**-c**_ refers to an existing context
-* _**-n**_ refers to the node name that should be a member of this context
-* _**-r as MESSAGING\_ONLY**_ refers to the role of this node within the context 
+* _**-c**_ refers to an existing replication group
+* _**-n**_ refers to the node name that should be a member of this replication group
+* _**-r as MESSAGING\_ONLY**_ refers to the role of this node within the replication group 
 
 _Optional parameters_
 
 * _**-S**_ if not supplied connects by default to [http://localhost:8024](http://localhost:8024). If supplied, it should be any node serving the _\_admin_ context 
 * _**-t**_  refers to the access token to authenticate at server
 
-C\) Axon Server EE provided REST API \(http:\[server\]:\[port\]/swagger-ui.html\) which offers the _context-rest-controller_ to help perform role maintenance operations
+C\) Axon Server EE provided REST API \(http:\[server\]:\[port\]/swagger-ui.html\) which offers the _replication-group-rest-controller_ to help perform role maintenance operations
+
+## Secondary nodes <a id="secondary-nodes"></a>
+
+> Axon Server version 4.4 and higher
+
+You can add nodes as secondary nodes to a replication group, if you want to limit the amount of data stored on primary nodes. 
+You could for example have fast (more expensive) storage on the primary nodes and less expensive storage on the secondary nodes. 
+Most activity will take place on recent data, and only when you need to access older data you have to go to the secondary nodes. 
+
+One you have defined secondary nodes for a replication group, this will apply for all contexts that are defined in the replication group.
```

Comment:
```suggestion
Once you have defined secondary nodes for a replication group, this will apply for all contexts that are defined in the replication group.
```